### DISCUSSION TITLE
改进：使用异步方式下载图片

### DISCUSSION THREAD

**Discussion by kzqj** [hect0x7/JMComic-Crawler-Python#255] [Ideas]
简单翻了代码，感觉下载图片用异步比多线程性能更好点，不过我看用的postman是request库的get，不知道能否对下载功能进行一下改进

**Comment by hect0x7** [comment]
主要是我其实也并不熟悉async的用法。如果你愿意提PR优化这一部分我会很乐意接受。

**Reply by kzqj** [reply]
如果之后有时间，我会尝试下pr
